README.md

```diff
@@ -2,52 +2,140 @@
 
 - [mcp-proxy](#mcp-proxy)
   - [About](#about)
+  - [1. stdio to SSE](#1-stdio-to-sse)
+    - [1.1 Configuration](#11-configuration)
+    - [1.2 Example usage](#12-example-usage)
+  - [2. SSE to stdio](#2-sse-to-stdio)
+    - [2.1 Configuration](#21-configuration)
+    - [2.2 Example usage](#22-example-usage)
   - [Installation](#installation)
-  - [Configuration](#configuration)
-    - [Claude Desktop Configuration](#claude-desktop-configuration)
-  - [Advanced Configuration](#advanced-configuration)
-    - [Environment Variables](#environment-variables)
+  - [Command line arguments](#command-line-arguments)
+  - [Testing](#testing)
 
 ## About
 
-Connect to MCP servers that run on SSE transport, or expose stdio servers as an SSE server using the MCP Proxy server.
+The `mcp-proxy` is a tool that lets you switch between server transports. There are two supported modes:
 
-## stdio to SSE
+1. stdio to SSE
+2. SSE to stdio
+
+## 1. stdio to SSE
+
+Run a proxy server from stdio that connects to a remote SSE server.
+
+This mode allows clients like Claude Desktop to communicate to a remote server over SSE even though it is not supported natively.
 
 ```mermaid
 graph LR
-    A["Claude Desktop"] <--> B["mcp-proxy"]
-    B <--> C["External MCP Server"]
+    A["Claude Desktop"] <--> |stdio| B["mcp-proxy"]
+    B <--> |SSE| C["External MCP Server"]
 
     style A fill:#ffe6f9,stroke:#333,color:black,stroke-width:2px
     style B fill:#e6e6ff,stroke:#333,color:black,stroke-width:2px
     style C fill:#e6ffe6,stroke:#333,color:black,stroke-width:2px
 ```
 
-> [!TIP]
-> As of now, Claude Desktop does not support MCP servers that run on SSE transport. This server is a workaround to enable the support.
+### 1.1 Configuration
+
+This mode requires the URL of the SSE server to be set. The `API_ACCESS_TOKEN` environment variable can be set to pass the access token to the remote server.
```

Comment:
Let's de-emphasize the authentication header since it is more advanced:

```suggestion
This mode requires passing the URL to the MCP Server SSE endpoint as the first argument to the program.
```
(Also seems pretty clear this will change as soon as the auth spec is added and it may go away entirely)